RELEASING.md

```diff
@@ -1,183 +1,201 @@
 # Releasing Google Cloud Ruby Client
 
-These instructions apply to every gem within the Google Cloud Ruby Client
-project.
+These instructions apply to every gem within the [Google Cloud Ruby
+Client](https://github.com/GoogleCloudPlatform/google-cloud-ruby) project.
 
 ## Releasing individual gems and meta-packages
 
 The Google Cloud Ruby Client project uses [semantic
-versioning](http://semver.org). Replace the `<prev_version>` and `<version>`
-placeholders shown in the examples below with the appropriate numbers, e.g.
-`0.1.0` and `0.2.0`. Replace the `<gem>` placeholder with the appropriate full
-name of the package, e.g. `google-cloud-datastore`.
+versioning](http://semver.org). Replace the `<version>` placeholder in the
+examples below with the appropriate number, e.g. `0.1.0`. Replace the `<gem>`
+placeholder with the appropriate full name of the gem, e.g.
+`google-cloud-datastore`.
 
 After all [pull
 requests](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pulls) for a
 release have been merged and all Kokoro and [Circle CI
 builds](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) are
 green, you may create a release as follows:
 
-1. If you haven't already, switch to the master branch, ensure that you have no
-   changes, and pull from origin.
+1. In root directory of the project, switch to the master branch, ensure that
+   you have no changes, and pull from the [project
+   repo](https://github.com/GoogleCloudPlatform/google-cloud-ruby).
 
     ```sh
     $ git checkout master
     $ git status
-    $ git pull <remote> master --rebase
+    $ git pull <remote> master
     ```
 
-2. Build the gem locally. (Depending on your environment, you may need to
-   `bundle exec` to rake commands; this will be shown.)
+1. Create and switch to a new branch with today's date in the name.
```

Comment:
I know opinions differ on this, but I definitely prefer keeping the actual numbers. It makes it a lot easier to review when we include text like "Repeat steps 4 through 13 if you are releasing multiple gems."

Reply:
We may have discussed this exact thing yesterday... 😆

Reply:
Sorry, it is so common to move steps around, using `1.` is the way to go for maintaining the file.

Reply:
As for ease of reviewing, definitely take advantage of the rendered version linked in the PR description above.